scripts/test-check-packages.sh

```diff
@@ -18,6 +18,6 @@ for d in test/packages/*/; do
   (
     cd $d
     elastic-package check -v
-    elastic-package test -v
+    elastic-package test -v -r xUnit
```

Comment:
Is it better to check against xUnit or Console? I wonder if there is a way to verify the generated XML test report.

Reply:
Good question. I changed it to `xUnit` because this script will be run by CI. In fact, I'm thinking of adding `--report-output file` to this and making necessary changes in the CI configuration so the test results are captured by Jenkins.

But I also realize that this script is simply called from `make test`, which could be run outside of a CI context too. So I guess we need to bubble up these two parameters (test report format and test report output) all the way up such that they can be overridden in a CI context to be xUnit+file but otherwise default to human+stdout.

Reply:
We can also output the test report into the test directory which is versioned. I wonder if the verification doesn't introduce too much complexity here as the file is not always the same (different execution time for tests). As the quick win you can just verify if the report file has been created?

On the other hand I'm ok with postponing it for the further improvement (not in scope of this PR). It's better to quickly enable it in integrations.

Reply:
My understanding is that with the appropriate changes to the CI pipeline Jenkins will be able to parse the xUnit formatted files and thus verify the results of the tests. I plan to include these changes in this PR shortly so we can see them in action in this PR's CI build jobs.